galley/pkg/config/analysis/analyzers/util/constants.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+const (
+	DefaultKubernetesDomain = "svc.cluster.local"
```

Comment:
Should we move fqdnPattern (from hosts.go) into this file as well? Even if it's prohibitively painful to make the regex use the const, I think we're less likely to run into "forgot to update one of them" errors if we keep them adjacent.